pkg/cluster/cluster.go

```diff
@@ -294,7 +294,8 @@ func (c *Cluster) createMachineRunArgs(machine *Machine, name string, i int) []s
 		"--tmpfs", "/run",
 		"--tmpfs", "/run/lock",
 		"--tmpfs", "/tmp:exec,mode=777",
-		"-v", "/sys/fs/cgroup:/sys/fs/cgroup:ro",
+		"--cgroupns", "host",
+		"-v", "/sys/fs/cgroup:/sys/fs/cgroup:rw",
```

Comment:
I think this needs to be a conditional setup as not all systems using footloose will have cgroups v2 in place. Or at least it needs to be made sure this does not break cgroups v1 based setups of docker. E.g. we (https://github.com/k0sproject/k0s) use footloose heavily without Docker Desktop e.g. in our CI where the runners are cgroups v1 based.